tools/check_consistency.py

```
@@ -0,0 +1,434 @@
+    report.findings.extend(vale_findings)
+    vale_dir = repos_dir / "vale-no-animal-violence"
+    if vale_dir.exists():
+        count = 0
+        for yml_file in vale_dir.rglob("*.yml"):
+            try:
+                with open(yml_file) as f:
+                    data = yaml.safe_load(f)
+                if data and "swap" in data:
+                    count += len(data["swap"])
+            except Exception:
+                pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)